book/src/libs/wasp/components/profile.md

```diff
@@ -0,0 +1,15 @@
+# WASP - Profile
+
+A **Profile** allows you to combine load from different generators. Each generator operates according to its own schedule, and you can even mix RPS and VU load types, as shown in [this example](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/wasp/examples/profiles/node_mixed_test.go).
+
+> [!NOTE]  
+> The `error` returned by the `.Run(true)` function of a `Profile` only indicates if any dashboard alerts were triggered.  
```

Comment:
I should probably make that comment more specific, because if you did use `.WithGrafana(opts *GrafanaOpts)` and your dashboard had alerts sets and any of them was triggered, we do return an error:
```
	if m.grafanaOpts.CheckDashboardAlertsAfterRun != "" {
		m.printDashboardLink()
		alerts, err := CheckDashboardAlerts(m.grafanaAPI, m.startTime, time.Now(), m.grafanaOpts.CheckDashboardAlertsAfterRun)
		if len(alerts) > 0 {
			log.Info().Msgf("Alerts found\n%s", grafana.FormatAlertsTable(alerts))
		}
		if err != nil {
			return m, err
		}
```

Reply:
added extra explanation